ipcc/build.rs

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#[cfg(target_os = "illumos")]
+static OXIDE_PLATFORM: &str = "/usr/platform/oxide/lib/amd64/";
```

Comment:
worth a comment explaining what this path is?

Reply:
Fixed in 77bd065.